src/Synchronization/Scheduler.php

```diff
@@ -30,7 +30,6 @@
 class Scheduler implements ContainerAwareInterface, FrameworkAwareInterface
 {
     use ContainerAwareTrait;
-
```

Comment:
This change seems unrelated.

Reply:
see: https://github.com/m-vo/contao-facebook-import/pull/33